public/app/plugins/datasource/cloud-monitoring/components/Fields.tsx

```diff
@@ -55,3 +31,57 @@ export const VariableQueryField: FunctionComponent<VariableQueryFieldProps> = ({
     </InlineField>
   );
 };
+
+export interface Props {
+  children: React.ReactNode;
+  tooltip?: PopoverContent;
+  label?: React.ReactNode;
+  className?: string;
+  noFillEnd?: boolean;
+  labelWidth?: number;
+  fillComponent?: React.ReactNode;
+}
+
+export const QueryEditorRow: FC<Props> = ({
+  children,
+  label,
+  tooltip,
+  noFillEnd,
+  fillComponent,
+  labelWidth = LABEL_WIDTH,
+}) => {
+  return (
+    <div className="gf-form">
+      {label && (
+        <InlineLabel width={labelWidth} tooltip={tooltip}>
+          {label}
+        </InlineLabel>
+      )}
+      <div
+        className={css`
+          margin-right: 4px;
```

Comment:
HorizontalGroup adds a 4px margin to all its children. ^ this margin between the HorizontalGroup and the row fill makes it look a bit better.